tests/conftest.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+import os
 import re
+import sys
 from datetime import timedelta
 
+sys.path.insert(0, os.path.abspath('..'))
```

Comment:
Hacking sys path is the best way to ensure the test works as expected. 

I think think it's many.
- ppl can git clone and start hacking right away without recreating the virtual env on every code base
- don't even need to have the package installed or run setup
- you don't need virtual env but you can still use it